bond_app/authentication.py

```diff
@@ -74,17 +77,21 @@ def require_user_info(self, request_state, token_info_fn=_token_info):
 
         token = auth_header_parts[1]
 
-        user_info = self.cache_api.get(key='access_token:' + token)
-        if user_info is None:
-            user_info = self._fetch_user_info(token, token_info_fn)
+        google_user_info = self.cache_api.get(key='access_token:' + token)
+        if google_user_info is None:
+            google_user_info = self._fetch_user_info(token, token_info_fn)
             # cache for 10 minutes or until token expires
-            expires_in = min([user_info.expires_in, self.config.max_token_life])
+            expires_in = min([google_user_info.expires_in, self.config.max_token_life])
             logging.debug("caching token %s for %s seconds", token, expires_in)
```

Comment:
I'm worried that we'd flood the logs if these were `info` level. As I understand it, we want this caching so that when users are running workflows on potentially many VMs, we don't slam Google or Sam trying to get user info. If these were all `info` statements, anytime a user ran a workflow it could just show flood the logs

Reply:
well, I think app engine is logging entries per request anyway, so no matter what, when a workflow is run, if it's banging on Bond, it's going to log (probably multiple) entries per request anyway.  I don't know how many log entries app engine is making for each request right now, but even if it's 2 or 3 entries per request now, I don't see harm in going to 5 or 6 or even 10 so long as the information logged is useful.  I think this is a big reason we've struggled in debugging Bond stuff in the past is just a lack of logging.

Reply:
this looks like a debug level message to me. We should not be inflating the message priority to get more verbosity out of our code, we should instead manipulate the logging level reported. If we want to see debug level statements in stack driver we should make sure debug log messages are emitted.